app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,9 @@
+<article class="d-flex w-75 p-3">
+  <div class="border border-5 me-3" style="width: 120px;">
+    <img class="img-fluid"src="/assets/user_avatar-svgrepo-com.svg" alt="USER AVATAR" srcset="">
+  </div>
```

Comment:
- [ ] Good job adding an image here @Nkaleth, Although your work is fine, we need to make sure we are rendering the appropriate picture for each user by using `user.photo` instead of an avatar that is the same for all.